src/StreamJsonRpc/LengthHeaderMessageHandler.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace StreamJsonRpc
+{
+    using System;
+    using System.Buffers;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.IO.Pipelines;
+    using System.Runtime.InteropServices;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft;
+    using Nerdbank.Streams;
+    using StreamJsonRpc.Protocol;
+
+    /// <summary>
+    /// A minimal header for each message that is simply declares content length.
```

Comment:
nit: I think you might be missing a comma in this comment

Reply:
Fixed. Thanks.